docs/add-ons/index.md

```diff
@@ -6,31 +6,31 @@ The framework currently provides support for the following add-ons:
-| [Cluster Autoscaler](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/cluster-autoscaler.md) | Deploys the standard cluster autoscaler into an EKS cluster. |
-| [Crossplane](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/crossplane.md) | Deploys Crossplane into an EKS cluster. |
-| [ExternalDNS](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/external-dns.md) | Deploys External DNS into an EKS cluster. |
-| [Fargate Fluent Bit](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/fargate-fluent-bit.md) | Adds Fluent Bit support for EKS Fargate |
-| [Karpenter](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/karpenter.md) | Deploys Karpenter into an EKS cluster. |
-| [Keda](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/keda.md) | Deploys Keda into an EKS cluster. |
-| [Kube State Metrics](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/kube-state-metrics.md) | Deploys Kube State Metrics into an EKS cluster. |
-| [Kubernetes Dashboard](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/kubernetes-dashboard.md) | Deploys Kubernetes Dashboard into an EKS cluster. |
-| [Metrics Server](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/metrics-server.md) | Deploys the Kubernetes Metrics Server into an EKS cluster. |
-| [Ingress Nginx](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/nginx.md) | Deploys the NGINX Ingress Controller into an EKS cluster. |
-| [Prometheus](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/prometheus.md) | Deploys Prometheus into an EKS cluster. |
-| [Spark K8S Operator](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/spark-on-k8s-operator.md) | Deploys the Kubernetes Operator for Apache Spark into an EKS cluster. |
-| [Tetrate Istio](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/tetrate-istio.md) | Deploys Tetrate Istio Distro into an EKS cluster. |
-| [Traefik](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/traefik.md) | Deploys Traefik Proxy into an EKS cluster.
-| [HashiCorp Vault](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/vault.md) | Deploys HashiCorp Vault into an EKS cluster.
-| [VPA](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/vpa.md) | Deploys the Vertical Pod Autoscaler into an EKS cluster. |
-| [YuniKorn](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/docs/add-ons/yunikorn.md) | Deploys Apache YuniKorn into an EKS cluster. |
+| [Agones](agones.md) | Deploys Agones into an EKS cluster. |
```

Comment:
I believe we want the full URL since this is coming from the mkdocs generated site pointing back to the individual addon REAMDEs

Reply:
Same as above. IMHO, this is correct.